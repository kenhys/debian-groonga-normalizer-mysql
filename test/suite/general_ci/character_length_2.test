register normalizers/mysql

normalize NormalizerMySQLGeneralCI "λογία"
