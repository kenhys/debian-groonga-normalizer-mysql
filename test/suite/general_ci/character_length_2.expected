register normalizers/mysql
[[0,0.0,0.0],true]
normalize NormalizerMySQLGeneralCI "λογία"
[[0,0.0,0.0],{"normalized":"ΛΟΓΙΑ","types":[],"checks":[]}]
